### ISSUE TITLE
Add a "Sort by Hot" or a "sort by active"

### ISSUE THREAD

**Issue by LunaWurx** [modrinth/knossos#479]
### Describe the solution you'd like

Add a "Sort by hot" filter. It sorts by most follows and downloads in the past hour/minute/day. 

### Describe alternatives you've considered

none, really

### Additional context

This shows mods that are popular in current times and/or ones that are becoming popular. This helps new projects to be discovered easier and older mods that are used less to be less dominant

**Comment by DrParadox7** [comment]
Yeah, this is a feature Modrinth lacks that is detrimental to the ease of use on the platform.
Right now users have to scroll through the most downloaded projects and scrutinize which projects are good and which are just old while new projects don't really get seen.

While having an option to sort through the past hour/minute/day is nice, as long as Modrinth matches the functionality of CurseForge it would suffice. 

I believe they just check for daily downloads but I could be wrong
![image](https://user-images.githubusercontent.com/47131096/183051354-9b84760f-3355-4c1d-83f8-eb12f9632280.png)
